### DISCUSSION TITLE
Create a donation page on the website

### DISCUSSION THREAD

**Discussion by freewarecamp** [friction2d/friction#82] [General]
Hello, I would like to say that a donation page is needed on the website to support the development of Friction. With as many payment method possible like PayPal, Payoneer, Bitcoin wallet address, Binance etc.

Thank you.

**Comment by rodlie** [comment]
Finding a way to fund the project should be a goal.

There are several ways to do this, I have currently not decided on anything yet.

**Reply by dejayc** [reply]
Please keep us up to date - I would love a way to support this software.

**Reply by AlmostMikhail** [reply]
I would support as well. Whatever I can
